.deploy/nais/app-preprod.yaml

```diff
@@ -39,10 +39,12 @@ spec:
   azure:
     application:
       enabled: true
-      allowAllUsers: true
+      allowAllUsers: false
```

Comment:
er usikker på om dette vil virke for infotrygd når de skal kalle oss. når denne settes til false er det kun brukere som er definert i groups som få lov til å hente token fra AAD, ref https://docs.nais.io/security/auth/azure-ad/access-policy/#groups
Jeg tenker det er godt nok med true her + TilgangskontrollService du har laget

Reply:
fikset det